boardgamegeek/utils.py

```diff
@@ -335,6 +335,10 @@ def request_and_parse_xml(requests_session, url, params=None, timeout=15, retrie
                         time.sleep(retry_delay)
                         retry_delay *= 1.5
                     continue
+            elif r.status_code == 404:
```

Comment:
Any idea if the legacy behaves the same way as the new api in regards to throttling the requests? i.e., does it throttle, does it also return 503 if too many requests are made, etc?

Reply:
This is likely one of the most risky modifications.
I needed this for the legacy API because it does not return any error message, only a 404 code.

Also, I cannot rely on the unit tests, because I'm not sure they behave consistently with the actual XMLAPI2, as far as the HTTP return code is cocnerned (for inexistant items).

I sure will test this also.